Description: Do not use the sphinxcontrib.napoleon sphinx theme
 This theme isn't available in Debian (yet), and I don't have the time to
 package it, so for now, I'm just disabling the use of it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-02-17

Index: python-kafka/docs/conf.py
===================================================================
--- python-kafka.orig/docs/conf.py
+++ python-kafka/docs/conf.py
@@ -32,7 +32,6 @@ extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.intersphinx',
     'sphinx.ext.viewcode',
-    'sphinxcontrib.napoleon',
 ]
 
 # Add any paths that contain templates here, relative to this directory.
